README.md

```diff
@@ -152,6 +129,7 @@ yarn site
 ```
 
```

Comment:
hey @bryceosterhaus, With the migration guide going to be a WIP process, we can add a category informing about the migration guide pointing to the documentation soon.

Something like:
```
## Migration Guides
Do you already have the Clay v2 components implemented in your project? These handy guides will help you perform the migration from Clay v2 to v3.

- [Migrate the Clay components from v2 to v3](https://next.clayui.com/docs/guides/migrating-from-v2-to-v3)
```